parlai/agents/transformer/modules.py

```diff
@@ -428,9 +429,9 @@ def forward_embedding(
             positions = (mask.cumsum(dim=1, dtype=torch.int64) - 1).clamp_(min=0)
         tensor = self.embeddings(input)
         if self.embeddings_scale:
-            tensor = tensor * np.sqrt(self.dim)
+            tensor = tensor * math.sqrt(self.dim)
```

Comment:
why this change? just curious

Reply:
Changing this back - some stuff you can't do when TorchScript scripting, but you *can* do it when TorchScript tracing, which we now do and has looser requirements